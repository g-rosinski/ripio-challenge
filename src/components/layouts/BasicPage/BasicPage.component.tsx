import React  from "react"
import { PageContainer } from "./BasicPage.styles"

interface BasicPageProps {
    children: React.ReactNode
}

export const BasicPage: React.FC<BasicPageProps> = ({children}) => {
    return (
        <PageContainer>
            { children }
        </PageContainer>
    )
}

export default BasicPage