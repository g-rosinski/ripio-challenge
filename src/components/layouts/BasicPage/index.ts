export * from './BasicPage.component'
export * from './BasicPage.styles'