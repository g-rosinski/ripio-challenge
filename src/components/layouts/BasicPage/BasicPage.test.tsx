import { render } from '@testing-library/react'
import React from 'react'
import BasicPage from './BasicPage.component'

describe('<BasicPage/>', () => {

    it('Se espera que se reenderice correctamente', () => {
        const {container} = render(<BasicPage><></></BasicPage>)

        expect(container).toMatchSnapshot()
    })

})