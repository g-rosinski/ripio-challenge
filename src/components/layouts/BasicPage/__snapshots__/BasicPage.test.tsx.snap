// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<BasicPage/> Se espera que se reenderice correctamente 1`] = `
<div>
  <div
    class="sc-bcXHqe dYjOst"
  />
</div>
`;
